src/School39.Landing/Pages/Home.razor

```diff
@@ -5,4 +5,5 @@
 <TopbarComponent/>
 <HeaderComponent/>
 <HeroComponent/>
+<ContactusComponent/>
 
```

Comment:
It is better to delte line9. There is nothing in there.

Reply:
code changed thanks for the reminder